framework/src/postprocessors/PseudoTimestep.C

```diff
@@ -0,0 +1,208 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// Navier-Stokes includes
+#include "PseudoTimestep.h"
+#include "FEProblemBase.h"
+#include "NonlinearSystemBase.h"
+#include "Transient.h"
+#include "Restartable.h"
+
+// MOOSE includes
+#include "libmesh/quadrature.h"
+#include "libmesh/enum_norm_type.h"
+#include "metaphysicl/raw_type.h"
+
+#include <algorithm>
+#include <limits>
+
+registerMooseObject("MooseApp", PseudoTimestep);
+
+InputParameters
+PseudoTimestep::validParams()
+{
+  InputParameters params = GeneralPostprocessor::validParams();
+
+  MooseEnum method("SER=1 EXP=2 RDM=3");
```

Comment:
it would be cool if you used `method.addDocumentation("SER", "Switched evolution relaxation");` etc, to document the cryptic Moose enum options. Maybe you could be even more verbose in the doc string...

Reply:
This `method.add` is quite neat. How do I check locally that the oxygen worked fine?